nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/import-from-registry/import-from-registry.component.ts

```diff
@@ -212,7 +212,8 @@ export class ImportFromRegistry extends CloseOnEscapeDialog implements OnInit {
     flowChanged(flowId: string): void {
         const registryId = this.importFromRegistryForm.get('registry')?.value;
         const bucketId = this.importFromRegistryForm.get('bucket')?.value;
-        this.loadVersions(registryId, bucketId, flowId);
+        const branch = this.importFromRegistryForm.get('branch')?.value;
+        this.loadVersions(registryId, bucketId, flowId, branch);
```

Comment:
It looks like there is a similar function for `bucketChanged` which then calls `loadFlows(registryId, bucketId)` and I think that would need to include the branch as well right?